client/question/components/LearnerViewDirective.js

```diff
@@ -886,7 +886,7 @@ tie.directive('learnerView', [function() {
             var feedbackParagraphs = feedback.getParagraphs();
             for (var i = 0; i < feedbackParagraphs.length; i++) {
               clearAllHighlights();
-              if (feedbackParagraphs[i].isErrorParagraph()) {
+              if (feedbackParagraphs[i].getErrorLineNumber()) {
```

Comment:
Is the error line number 0 or 1 indexed? If the former then what happens if it is 0?

In any case, might want to say "!== null" instead if that's really what you're checking.